modules/iam-types/src/main/scala/ch/epfl/bluebrain/nexus/commons/iam/types/acls/AccessControl.scala

```diff
@@ -0,0 +1,8 @@
+package ch.epfl.bluebrain.nexus.commons.iam.types.acls
```

Comment:
let's use the base package `ch.epfl.bluebrain.nexus.commons.iam`, no types suffix.